src/igvfd/searches/configs/construct_library_set.py

```diff
@@ -57,6 +90,21 @@ def construct_library_set():
                     'file_set_type',
                     'selection_criteria',
                     'scope',
+                    'associated_phenotypes.term_name',
+                    'tiling_modality',
+                    'guide_type',
+                    'targeted_genes',
```

Comment:
The path here is mismatched from the above defined facet (line 25)

Reply:
You are correct, also it is not the right property - should have been small_scale_gene_list instead. Thanks for catching that.